setup.py

```diff
@@ -53,8 +53,8 @@
         'Babel >= 0.9.4',
         'Biryani[datetimeconv] >= 0.10.4',
         'numpy >= 1.11',
-        'OpenFisca-Core >= 5.0.1, < 6.0',
-        'OpenFisca-Parsers >= 1.0.1, < 2.0',
+        'OpenFisca-Core >= 5.0.1, < 8.0',
+        'OpenFisca-Parsers >= 1.0.2, < 2.0',
```

Comment:
To adapt is [this PR](https://github.com/openfisca/openfisca-core/pull/472) is merged before.